data/templates/css/wikimedia.scss

```diff
@@ -572,4 +572,8 @@ $spacing-elements: 30px;
             text-align: left;
         }
     }
+    html {
+        -webkit-text-size-adjust:none;
+        font-size:100%
+}
```

Comment:
Indentation of }

Reply:
_This is tricky:_
Currently there is code to rewrite the URIs from the actual HTML content to be server relative but for this case we would need to process the css and rewrite the contained URIs.

Can we change the icon for the infobox, and change it from the +- to an arrow without introducing an icon? That should be easier for now We can come back to it (read as: I would not like to block on it for now)